Makefile

```diff
@@ -17,7 +17,7 @@ else
 endif
 
 ifeq ($(shell uname -p | sed -r 's/.*(86|armel|armhf|aarch64|ppc64le|s390x).*/golang/'), golang)
-	GO_C = golang-1.8
+	GO_C = golang-1.9
```

Comment:
all of those architectures are now supported by gc - do we need to keep the gccgo bits hanging around?

Reply:
Ahh, right you are. I think gccgo bits can all be removed. I'll do so.